src/main.ts

```diff
@@ -1,13 +1,13 @@
 import {
+  getInput,
   getState,
   saveState,
-  getInput,
   setFailed,
   setOutput,
   setSecret
 } from '@actions/core'
-import {platform} from 'os'
 import {writeFileSync} from 'fs'
+import {platform} from 'os'
```

Comment:
I assume your IDE moved these to keep things in alphabetical order. Is there a linter rule we can add to enforce this going forward?